docs/perl-ads.json

```diff
@@ -1,58 +1,72 @@
 [
-  {
-    "title": "Call for Participation - Perl Community Conference, Summer 2025",
-    "description": "Hybrid / Austin, Texas July 3-4, 2025",
-    "link": "https://www.papercall.io/perlcommunity",
-    "end": "2025-07-05",
-    "publisher": "perl-community"
-  },
-  {
-    "title": "The Perl and Raku Conference 2025",
-    "description": "Greenville, South Carolina - June 27-29",
-    "link": "https://tprc.us/",
-    "end": "2025-06-29",
-    "publisher": "tprf"
-  },
-  {
-    "title": "From Code to Community",
-    "description": "Sponsoring The Perl and Raku Conference 2025",
-    "link": "https://www.perl.com/article/from-code-to-community-sponsoring-tprc-2025/",
-    "end": "2025-06-29",
-    "publisher": "tprf"
-  },
-  {
-    "title": "Vienna.pm BBQ",
-    "description": "The yearly BBQ by Vienna.pm on Donauinsel",
-    "link": "https://vienna.pm.org/kamelgrillen.html",
-    "end": "2025-06-13",
-    "publisher": "vienna-pm"
-  },
-  {
-    "title": "Perl x Open Food Facts Hackathon",
-    "description": "Paris, France - May 24-25",
-    "link": "https://forum.openfoodfacts.org/t/hackathon-perl-open-food-facts-in-paris-2025/1936",
-    "end": "2025-05-26",
-    "publisher": "open-foods-facts"
-  },
-  {
-    "title": "CPAN Security Tip #1/3",
-    "description": "Add a SECURITY.md policy to your favorite CPAN dists!",
-    "link": "https://security.metacpan.org/docs/guides/security-policy-for-authors.html",
-    "end": "2026-04-30",
-    "publisher": "cpansec"
-  },
-  {
-    "title": "CPAN Security Tip #2/3",
-    "description": "Make sure none of your favourite CPAN dists are unmaintained!",
-    "link": "https://neilb.org/2013/07/24/adopt-a-module.html",
-    "end": "2026-04-30",
-    "publisher": "cpansec"
-  },
-  {
-    "title": "CPAN Security Tip #3/3",
-    "description": "Get help with reporting or fixing a security issue on CPAN!",
-    "link": "https://security.metacpan.org/docs/report.html",
-    "end": "2026-04-30",
-    "publisher": "cpansec"
-  }
-]
\ No newline at end of file
+   {
+      "description" : "Paris, France - May 24-25",
+      "end" : "2025-05-26",
+      "link" : "https://forum.openfoodfacts.org/t/hackathon-perl-open-food-facts-in-paris-2025/1936",
+      "publisher" : "open-foods-facts",
+      "title" : "Perl x Open Food Facts Hackathon"
+   },
+   {
+      "description" : "The yearly BBQ by Vienna.pm on Donauinsel",
+      "end" : "2025-06-13",
+      "link" : "https://vienna.pm.org/kamelgrillen.html",
+      "publisher" : "vienna-pm",
+      "title" : "Vienna.pm BBQ"
+   },
+   {
+      "description" : "Posted: 2025-05-15",
+      "end" : "2025-06-19",
+      "link" : "https://jobs.perl.org/job/22043",
+      "publisher" : "PerlToolsTeam",
```

Comment:
"publisher" : "perl-jobs",

Reply:
Good call! I figured I'd try "jobs.perl.org" as the publisher since it's descriptive and doesn't require us to make up a new name.

Reply:
It's either published by Ask, or ' Smoothwall by QORIA', right?

Not to get all post modern or anything, but it really depends a lot of what you mean by "publisher"

Reply:
I guess it's really "source of the ad". It shows where the Ad Server got this ad from. We don't care about who created the job ad. We only care that this ad came from the jobs site.